source/import-export.txt

```diff
@@ -49,20 +49,30 @@ that it is formatted correctly.
        name: JSON
```

Comment:
I can't comment on it directly, but at the top of the "Export" section on this page we have a blurb which states: "If you specify a filter in the query bar prior to export, Compass can optionally only export documents which match the specified query."

I don't think this is 100% true any more, since you can modify the filter prior to export. Can we modify that blurb to make it consistent with the new behavior?

Reply:
Removed "...in the :ref:`query bar <compass-query-bar>` prior to export" since it's misleading. Let me know if you had more changes in mind.